### PR TITLE
s/lelandrichardson/airbnb/

### PR DESCRIPTION
**Pull request by spikebrehm**
Now that we've moved the repo over, updated all references of
`lelandrichardson` to be `airbnb`.

to: @lelandrichardson @ljharb

### CODE REVIEW

**Comment by ljharb** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -262,7 +262,7 @@ Markers are draggable, and emit continuous drag events to update other UI during
 The API of this Map has been built with the intention of it being able to utilize the [Animated API](https://facebook.github.io/react-native/docs/animated.html).
 
 In order to get this to work, you will need to modify the `AnimatedImplementation.js` file in the
-source of react-native with [this one](https://gist.github.com/lelandrichardson/c0d938e02301f9294465).
+source of react-native with [this one](https://gist.github.com/airbnb/c0d938e02301f9294465).
```
Comment (jrichardlai) [inline README.md]: this should be lelandrichardson

Comment (ljharb) [inline README.md]: whoops, good catch

**Comment by spikebrehm** [comment]
@jrichardlai nice catch! updated

**Pull request merged**